docs/testing.rst

```diff
@@ -0,0 +1,29 @@
+.. _testing:
+
+======================
+Tests for Page Objects
+======================
+
+``web-poet`` provides :ref:`tools for testing page objects
+<web-poet:web-poet-testing>`. ``scrapy-poet`` projects can use a Scrapy command
+to easily generate tests::
+
+    scrapy savefixture my_project.po.MyItemPage http://example.com/page1.html
```

Comment:
```suggestion
    scrapy savefixture my_project.pages.MyItemPage 'https://example.com/page1.html'
```

see also what we needed to do for Scrapy docs: https://docs.scrapy.org/en/latest/intro/tutorial.html#extracting-data